.chloggen/featuregate_cli_python.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note:  remove featuregate `EnablePythonAutoInstrumentationSupport`. Use command line flag `--enable-python-instrumentation` instead
```

Comment:
What was the `flag` what we are removing?

Reply:
Should be `operator.autoinstrumentation.python`